c10/core/DispatchKey.h

```diff
@@ -444,6 +444,12 @@ enum class DispatchKey : uint16_t {
   Autograd,
   CompositeImplicitAutograd, // registered at
   // build/aten/src/ATen/RegisterCompositeImplicitAutograd.cpp
+
+  // Note: [Disjoint Alias Keyset] Is joint from all the other alias keys
```

Comment:
Nit: "The alias keyset for CompositeImplicitBatched is disjoint from all other alias keysets" .. and so precedence order doesn't matter

Reply:
done